Release/src/websockets/client/ws_client_wspp.cpp

```diff
@@ -694,6 +724,7 @@ class wspp_callback_client : public websocket_client_callback_impl,
             case websocket_message_type::binary_message:
                 client.send(this_client->m_con, sp_allocated.get(), length, websocketpp::frame::opcode::binary, ec);
                 break;
+            case websocket_message_type::ping: client.ping(this_client->m_con, "", ec); break;
```

Comment:
Use message data, if added as suggested above.

Reply:
Added
As websocketpp doesn't have a `ping(void*, length)` but just a `ping(std::string param)`, we need to cast